test/test_fx.py

```diff
@@ -2258,6 +2259,251 @@ def test_get_torch_func_signature_exhaustive(self, device, dtype, op):
         except Exception as e:
             assert op.name in known_no_schema
 
+
+class TestFunctionalTracing(JitTestCase):
+    IGNORE_FUNCS = ('has_torch_function', 'has_torch_function_unary',
+                    'has_torch_function_variadic', 'handle_torch_function',
+                    'boolean_dispatch')
+    TO_PATCH = {'has_torch_function': None,
+                'has_torch_function_unary': None,
+                'has_torch_function_variadic': None}
+
+    BUILT_IN_FUNC = (AssertionError, '')
```

Comment:
Is there no exception message for htis one?

Reply:
Yes